OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -1607,6 +1644,43 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 					}
 				}
 
+				// Detonates replaces DemoTruck and changes MadTank
+				if (engineVersion < 20150629)
```

Comment:
Nobody noticed that the bigger issue is that AttackCharge will break this class, so I'll have to find another solution or fix my current solution.

On 2015-07-15, at 2:28 PM, Pavel Penev <notifications@github.com> wrote:

> In OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs:
> 
> > @@ -1607,6 +1644,43 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
> >  					}
> >  				}
> >  
> > +				// Detonates replaces DemoTruck and changes MadTank
> > +				if (engineVersion < 20150629)
> This should have its date updated.
> 
> ―
> Reply to this email directly or view it on GitHub.
>